src/Castle.Core.Tests/BasePEVerifyTestCase.cs

```diff
@@ -143,7 +143,8 @@ public void RunPEVerifyOnGeneratedAssembly(string assemblyPath)
 
 			var result = process.ExitCode + " code ";
 
-			Console.WriteLine(GetType().FullName + ": " + result);
+			// Clutters logs.
+			// Console.WriteLine(GetType().FullName + ": " + result);
```

Comment:
I guess the console was captured by NUnit and went into the results file. I think we can just remove the line since we've got an `Assert.Fail` if the return code is non zero.